source/2020-05-01-the-ember-times-issue-146.md

```diff
@@ -46,14 +47,13 @@ Interested in writing a post about an a11y topic such as screenreaders, WCAG, or
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Call for contributors: Shipping the Ember website(s) redesign 🎨🐹](https://twitter.com/melaniersumner/status/1254497875062673411)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Early last year the community came together to [discuss the future of the design strategy of official Ember sites](https://twitter.com/emberjs/status/1105157291806883840). After plenty of time and work investments from individual contributors, companies and the Ember Core team, the redesign for the [original RFC](https://emberjs.github.io/rfcs/0425-website-redesign.html) had [shipped successfully](https://twitter.com/emberjs/status/1230912205631213569) for the main site of the project: [emberjs.com](https://emberjs.com)!
```

Comment:
What do you think about changing the redirect link for "discuss the future" from Twitter to the actual RFC, so that there's one less jump to do? I think we can also shorten the second half of the 2nd sentence.
```suggestion
Early last year, the community came together to [discuss the future of the design strategy of official Ember sites](https://github.com/emberjs/rfcs/pull/425). After plenty of time and work investments from individual contributors, companies and the Ember Core team, the redesign for the main site [emberjs.com](https://emberjs.com) [shipped successfully](https://twitter.com/emberjs/status/1230912205631213569)!
```

Reply:
That sounds great!